src/html/_script_loader.html.template

```diff
@@ -1,3 +1,11 @@
+<script <% if (!useWDS) { %>crossorigin="use-credentials"<% } %>>
```

Comment:
Note this technically makes a change to add the `crossorigin` attribute to the demo and cast templates, but I assume that's not a big deal?